core/src/modules/module_maker.py

```diff
@@ -1,39 +1,40 @@
+"""Generate the code for a Nextsim module class source file."""
+
 def denamespace(nname):
-    """Returns the last element of the name, without any of the qualifying
-    namespaces."""
+    """Return the last element of the name, without any of the qualifying namespaces."""
     return nname.split(":")[-1]
 
 def generator(module_class_name, fq_interface_name, fq_impl_names, help_names):
-    """Generates the text for a module support file."""
+    """Generate the text for a module support file."""
```

Comment:
Is there a reason the prescriptive second person form (generate), rather than the descriptive third person form (generates)? Admittedly I got this from writing javadoc comments many years ago, rather than any Python standard, but I feel it makes more sense to me.

Reply:
It comes from `D401 First line of docstring should be in imperative mood`. We could ignore this rule if preferred?